content/about/use-case/_index.fr.md

```diff
@@ -5,24 +5,104 @@ description: Applications et axes directeurs
+ - [études d'exploitation](#%C3%A9tudes-dexploitation)
+ - [gestion d'infrastructure](#gestion-dinfrastructure)
+ - [production de sillons de dernière minute](#sillons-de-derni%C3%A8re-minute)
 
-![OSRD presentation](osrd_presentation.png)
 
-Notre mission est de fournir un outil de simulation qui peut être utilisé dans **toutes les étapes de la phase de conception** (planification des horaires et études d’exploitation), sur une **échelle géographique nationale** (comprenant le réseau français) et avec une **granulométrie variable des détails**. De plus, dans la phase opérationnelle, le projet OSRD propose un service pour tracer des **sillons de dernières minutes** (tranche d’espace-temps durant laquelle l’infrastructure est affectée à la circulation d’un train spécifique) dans la capacité résiduelle théorique ou réelle du réseau ferroviaire, une fois les autres circulations ont été déjà planifiées.
+![produits OSRD](osrd_product.fr.png)
+
+
+## Conception des plans de transport
+
+La conception de plans de transport consiste à formuler une réponse[^plan-exploitation] aux
+schémas de dessertes[^schema-desserte]. OSRD fournit déjà un certain nombre de fonctionnés utiles à cette fin :
+ - base d'infrastructure ferroviaires à l'échelle microscopique
+ - base de donnée de matériel roulant
+ - possibilité de modifier l'infrastructure
+ - visualisation de grilles horaires
+ - application de différents types de marges sur les sillons, y compris une
+   marge économique, qui minimise l'énergie consommée
+
+Les perspectives futures comprennent notament :
+ - la détection automatique de conflits[^conflit]
+ - synthèse d'infrastructure macroscopique à partir d'infrastructure microscopique
+ - visualisation schématique du plan de transport
+ - module d'optimisation de la capacité en ligne, d'occupation de la voie en gare et de roulement matériel
+
+[^plan-exploitation]: Sous forme de plan d'exploitation
+[^schema-desserte]: Un schéma de desserte est une demande d'offre de transport, qui comprend:
+    - la représentation des trains demandés
+    - leur itinéraire
+    - les arrêts à desservir
+
+[^conflit]: Un conflit se produit lorsque la circulation d'un train en gêne un autre
+
+
+## Études d'exploitation
+
+Les études d'exploitation permettent d'évaluer la capacité de l'infrastructure à
+répondre à la demande et de la mettre en cohérence avec les besoins de transport.
+Elles permettent de prendre des décisions d'investissements d'infrastructure publique éclairées.
+
+OSRD permet de répondre à ce besoin en :
+ - fournissant une base d'infrastructure ferroviaires à l'échelle microscopique
+ - fournissant une base de donnée de matériel roulant
+ - donnant la possibilité de modifier l'infrastructure existante et ajouter des installations
+ - permettant de visualiser les grilles horaires
+ - permettant d'appliquer différents types de marges sur les sillons, dont une
+   répartition économique, qui minimise l'énergie consommée.
```

Comment:
Why is there an ending period to this sentence? Please review all lists according to the general comment I wrote at the higher level.